### ISSUE TITLE
Add Qwt

### ISSUE THREAD

**Issue by traversaro** [Microsoft/vcpkg#367]
It would be great to have a port for the Qwt ( Qt Widgets for Technical Applications  ) library ( http://qwt.sourceforge.net/ ).

**Comment by traversaro** [comment]
Qwt uses a `qmake` based build system, so I guess that a first step would be to add `qmake` support in `vcpkg_find_acquire_program`.

**Comment by traversaro** [comment]
Actually `qmake` should be provided by the `qt5` port, similarly to how `protoc` is provided by the `protobuf` port.

**Comment by ras0219-msft** [comment]
I think the route of find_acquire_program is better for "non-deployable" utility programs, such as sed, python2 (when used as a build tool), qmake, cmake, etc.

The argument here is that building something from source is most critical when it will be injected into your application's address space and you need to "support" it. For tools that you use during the build but do not deploy, there is significantly less gain in building it locally compared to a precompiled download.

Tools that involve a deployed runtime such as protoc are an interesting intersection that seems best to solve case-by-case.

**Comment by traversaro** [comment]
I was preparing to reply to your comment @ras0219-msft , but then I saw that the PR https://github.com/Microsoft/vcpkg/pull/494 was merged. Replying anyway, motivating some design choices behind that PR. 

> For tools that you use during the build but do not deploy, there is significantly less gain in building it locally compared to a precompiled download.

The main problem with `qmake` is that the `qt5` port already installs it in `<install_prefix>/tools` as the `protobuf` port installs `protoc`, and consequently, as all qmake-based projects I am aware of also depend on qt5 as a library, it is drastically simpler to use the `qmake` already available in the system. 

Furthermore, no upstream binaries for the `qmake` as itself are available, but only binaries for the complete qt5 set of libraries and tools are available, and that would mean several MBs of additional download just to download a binary version of qmake. 

Note also that `qmake` is always provided in all package managers together with `qt5` and all its tools. Sincerely, I would really prefer to avoid being the first package manager to test the situation in which the `qmake` can come from a version of `qt5` different from all the `qt5` libraries and tools used by the project. 

This three points were for me sufficient to prefer to use the `qmake` version installed by the `qt5` port. 
Note that at the moment the `qt5` port install the tools that it compile, and this would be a problem for crosscompiling on ARM, but I consider this a problem of the `qt5` port rather then of all the projects depending on `qt5` for its tools.
